platform/admin/ui/src/main/webapp/js/views/Alerts.view.js

```diff
@@ -60,7 +69,7 @@ define([
                         json.level = 'danger';
                         break;
                     default:
-                        json.level = 'info';
+                        json.level = 'danger';
```

Comment:
I added this switch in the original pr for the admin ui alerting and didn't account for the configuration dialog alerts. This now accounts for both. I will add a manual testing step to verify that the original admin-ui alerting is still correct.